rho/clicommands.py

```diff
@@ -563,12 +497,29 @@ def _do_command(self):
                   "Please use --reset with profile first.")
             sys.exit(1)
 
-        cmd_string = 'ansible-playbook rho_playbook.yml -i data/'\
-                     + profile + '_hosts ' + '-v -f ' + forks
+        if facts == ['default']:
+            facts_to_collect = 'default'
+        elif os.path.isfile(facts[0]):
+            facts_to_collect = _read_in_file(facts[0])
+        else:
+            assert isinstance(facts, list)
+            facts_to_collect = facts
+
```

Comment:
Probably need to add some code to do validation on the fact collection list provided. I will open an issue for this capability.